### DISCUSSION TITLE
Markdown: Links in details `<summary>` aren't properly rendered

### DISCUSSION THREAD

**Discussion by nschloe** [community/community#60120] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

MWE

```markdown
[a](b)

<b>[a](b)</b>

<details>
<summary>[a](b)</summary>
</details>
```

> [a](b)
>
> <b>[a](b)</b>
>
> <details>
> <summary>[a](b)</summary>
> </details>

It does work with `<a href=...>`, but it should also work with the simplified syntax.

**Comment by leonden** [comment]
Interesting post! Unfortunately, as far as I know, in this particular edge case, it's not possible to use the normal Markdown syntax.

**Reply by nschloe** [reply]
Yeah, that's why I filed it as a bug.
